it/controllers/TradingNameResolverControllerISpec.scala

```diff
@@ -18,15 +18,15 @@ package controllers
 
 import common.enums.VatRegStatus
 import itutil.ControllerISpec
-import models.api.{Individual, UkCompany, VatScheme}
+import models.api.{Individual, Partnership, UkCompany, VatScheme}
 import models.{ApplicantDetails, TradingDetails}
 import play.api.http.HeaderNames
 import play.api.libs.json.Json
 import play.api.test.Helpers._
 
 class TradingNameResolverControllerISpec extends ControllerISpec {
-  "Party type resolver" should {
-    "return SEE_OTHER and redirects to /former-name" in new Setup {
+  "Trading name page resolver" should {
+    "return SEE_OTHER and redirects to /former-name for Indivitual" in new Setup {
```

Comment:
I don't think it redirects to that url, same thing for all 3 tests in the spec. Just the test string needs updating.